### ISSUE TITLE
List by Keywords / Category

### ISSUE THREAD

**Issue by boldt** [mrzapp/nextcloud-cookbook#188]
The old version used the "Keywords" to group the recipes. Because of this, I organized all my recipes by Keywords. The current version uses the "Category". Unfortunately, I cannot find a way to filter my recipes by a keywords any more. I really like the new category view, but there are use cases, where the keywords are still very useful:

* List all recipes with the keyword "barbeque", independent from their category. 
    * The "burgers" will be in the category "Main dish",
    * The "potato salad" will be in the Category "sides", and 
    * The "burger sauce" will be in the category sauces.
* List all recipes with the keyword "gluten free", independent from their category

I propose an additional filter to switch between "Keywords" and  "Category" view.

**Comment by mrzapp** [comment]
I agree, we're missing the keywords as explicit UI. As I suggested before, @mcorteel should we provide a "tag cloud" somewhere in the interface? I think it would fit well underneath the title in the overview, like this:

![image](https://user-images.githubusercontent.com/2747428/78547862-0c4e1b80-7800-11ea-8133-82261bd94407.png)

What do you guys think?

**Comment by hitam4450** [comment]
I liked the "old" keywords system much better compared the existing categories!
However, refining the current system would mot advisable !

**Comment by boldt** [comment]
Under the title is a good idea. 

* **Case 1**: I click on the category "all recipies". The I click on the tag "barbeque". 
    * I'll see all barbeque recipies, independent from their category
* **Case 2**: I click on "main dishes". Then I click on the tag "gluten free"
    * I'll see all gluten free main dishes
* **Case 3**: I click on the category "all recipies". The I click on the tag "barbeque". Then on the tag "Gluten free"
    * I'll see all gluten free recipies for barbeque - I recomment an _AND_, not and OR_ to combine tags


then on "gluten free". 

The I click on "barbeque". I see all barbeque recipies

Will this be an

**Comment by mrzapp** [comment]
@boldt case 3 just seems like an extended specification of case 2, unless I'm mistaken. Either way, I support using an AND operator, as well as keeping the category scope.

**Comment by seyfeb** [comment]
This issue can be closed 🎊
